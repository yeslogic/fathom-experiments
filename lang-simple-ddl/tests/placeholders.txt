def test-placeholder : _ :=
  33 : Int64;
