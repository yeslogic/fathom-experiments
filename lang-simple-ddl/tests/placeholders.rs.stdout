const TEST_PLACEHOLDER: i64 = 33;
